airgun/views/task.py

```diff
@@ -17,8 +17,8 @@
 
 class TaskReadOnlyEntry(ReadOnlyEntry):
     BASE_LOCATOR = (
-        "//span[contains(., '{}') and contains(@class, 'param-name')]"
-        "/following-sibling::span"
+        "//span[contains(., '{}') and contains(@class, 'list-group-item-heading')]//parent::div"
```

Comment:
Just to let you know, double slash before `parent::div` is not necessary. Single slash would be enough. I don't think it has any practical implications, so not a blocker.